scrapers/ga/events.py

```diff
@@ -10,6 +13,19 @@ class GAEventScraper(Scraper):
     #  PYTHONPATH=scrapers poetry run os-update ga events --scrape start=YYYY-mm-dd
     tz = pytz.timezone("US/Eastern")
 
+    def get_key(timestamp):
```

Comment:
nit: typically these would live outside the class so they didn't need the unusual calling pattern